index.html

```diff
@@ -49,12 +50,26 @@ <h1>New Entry</h1>
             <textarea name="notes" id="notes" class="block input-font" required></textarea>
           </div>
           <div class="column-full text-right">
-            <button type="submit" class="save-button">SAVE</button>
+            <button type="submit" class="save-button hover">SAVE</button>
           </div>
         </main>
       </form>
     </div>
+    <div data-view="entries" class="container gray hidden" id="entries">
+      <div class="row">
+        <div class="column-full flex align-center justify-between">
+          <h1>Entries</h1>
+          <a href="#" class="new-button text-center flex justify-center align-center" id="newButton">NEW</a>
+        </div>
+      </div>
+      <ul id="entriesList">
+        <li class="row no-entries text-center" id="noClass">
+          <p class="column-full p-no-entry">No entries have been recorded.</p>
+        </li>
```

Comment:
This shouldn't be an `li`, it's not an entry in this list. I would just move this `p` element to be a sibling of the `ul`